configs/apps/fv3/spack.yaml

```diff
@@ -0,0 +1,12 @@
+spack:
+  concretization: separately
+  view: false
+  include:
+  - config.yaml
+  - packages.yaml
+  - modules.yaml
+  - site.yaml
+  - ${SPACK_STACK_DIR}/configs/repos/repos.yaml
+
+  specs:
+  - fv3-bundle-env
```

Comment:
There is a bit of confusion around the meaning of these environments. `fv3-bundle-env` is an application that uses JEDI with the FV3 dycore (and for now also with NASA GEOS). `ufs-bundle-env` is an application that uses JEDI with the entire UFS, not just the dycore.

Reply:
Yes, I didn't really understand what these bundles were. What do you think a good name would be? Just name it `fv3-bundle` and `ufs-bundle`? Or `jedi-fv3-bundle`...?

Reply:
I think the names of the bundles are ok, but we should call the apps `fv3-jedi` or `jedi-fv3` etc. Let's not worry about this now, we can rename them in a follow-up PR. There are many more of these: JEDI with soca, um, mpas, ...